tests/libssh/ssh.go

```diff
@@ -35,14 +35,14 @@ func DumpPrivateKey(privateKey *ecdsa.PrivateKey, file string) error {
 		Type:  "EC PRIVATE KEY",
 		Bytes: privateKeyBytes,
 	}
-	f, err := os.OpenFile(file, os.O_CREATE|os.O_WRONLY, 0600)
+	const filePermissions = 0o600
```

Comment:
A more meaningful name for the 
```suggestion
	const ownerOnlyReadable = 0o600
```